### PR TITLE
reduce webmail image-layers/sizes

### PR DESCRIPTION
**Pull request by HorayNarea**
it's logically the same as before, just less disk-space

### CODE REVIEW

File: webmails/rainloop/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
- && unzip *.zip \
+ && unzip -q *.zip \
  && rm -f *.zip \
  && rm -rf data/ \
  && find . -type d -exec chmod 755 {} \; \
  && find . -type f -exec chmod 644 {} \; \
- && chown -R www-data: *
+ && chown -R www-data: * \
+ && apt-get purge -y unzip \
+ && rm -rf /var/lib/apt/lists
```
Comment (curry684) [inline webmails/rainloop/Dockerfile]: If you want to make it as lean as possible:
`    && rm -rf /tmp/* /usr/share/doc/* /usr/share/man/* /var/lib/apt/lists/*`

File: webmails/roundcube/Dockerfile [hunk 2]
```diff
@@ -17,7 +15,8 @@ RUN rm -rf /var/www/html/ \
  && cd html \
  && rm -rf CHANGELOG INSTALL LICENSE README.md UPGRADING composer.json-dist installer \
  && sed -i 's,mod_php5.c,mod_php7.c,g' .htaccess \
- && chown -R www-data: logs temp
+ && chown -R www-data: logs temp \
+ && rm -rf /var/lib/apt/lists
```
Comment (curry684) [inline webmails/roundcube/Dockerfile]: `    && rm -rf /tmp/* /usr/share/doc/* /usr/share/man/* /var/lib/apt/lists/*`

File: webmails/rainloop/Dockerfile (hunk 1 above)
Comment (HorayNarea) [reply webmails/rainloop/Dockerfile]: In theory you are right but in practice man-pages don't get installed in debian-slim and the docs are only ~200KB

Comment (curry684) [reply webmails/rainloop/Dockerfile]: Like I said, no major difference, but if you're optimized for size why leave 200KB around?   😉 

Also in my experience `/tmp` can be a nasty one at times so I habitually clean all these folders up at the end of a `RUN`.

**Comment by kaiyou** [comment]
It looks okay, ill try to build and approve later today

File: webmails/rainloop/Dockerfile (hunk 1 above)
Comment (kaiyou) [reply webmails/rainloop/Dockerfile]: This could be further improved but currently none of these weighs more than a few kb.

**Pull request merged**